src/mappings/mappingSystemTokenHandlers.ts

```diff
@@ -22,4 +22,18 @@ export async function systemTokenTransferEvent(event: SubstrateEvent): Promise<v
     record.success = checkIfExtrinsicExecuteSuccess(event.extrinsic)
 
     await record.save();
+}
+
+export async function systemTokenTransferCall(extrinsic: SubstrateExtrinsic): Promise<void> {
+    const blockNumber = (extrinsic.block.block.header.number as Compact<BlockNumber>).toNumber();
+    let record = new SystemTokenTransfer(blockNumber.toString() + '-' + extrinsic.idx.toString());
+    record.from = extrinsic?.extrinsic?.signer?.toString();
+    record.to = extrinsic.extrinsic.args[0].toString();
+    record.txHash = extrinsic.extrinsic.hash.toString();
+    record.amount = (extrinsic.extrinsic.args[1] as Balance).toBigInt();
```

Comment:
Do we guarantee the `amount` field is always the second indexed item? If so, seems good to me, but this stood out when reading.